docs/BOOTING.md

```diff
@@ -135,7 +135,7 @@ in grub.cfg with graphical GRUB menu to get the device to boot again.
     11. `eve_install_skip_rootfs` - do not install rootfs partition onto device. May be selected from graphical GRUB menu.
     12. `eve_install_skip_zfs_checks` - install zfs by skipping minimum requirement checks.
     13. `eve_install_zfs_with_raid_level` - Sets raid level for zfs storage. Valid values are none,raid1,raid5,raid6. Default value is none. This option also applied for the first boot of a live image to prepare zfs persist pool instead of ext4.
-    14. `eve_install_kubevirt_etcd_sizeGB` - Size in GB of the etcd-storage zvol.  Defaults to 10GB.
+    14. `eve_install_k3s_etcd_sizeGB` - Size in GB of the etcd-storage zvol.  Defaults to 10GB.
```

Comment:
Though this comment is not relevant to this PR. With ZKS coming in, may be we need to bump up the default size of etcd storage ? @andrewd-zededa